style.css

```
@@ -75,18 +119,112 @@ body {
+    display: none;
+    position: fixed;
+    top: 0;
+    left: 0;
+    width: 100vw;
+    height: 100vh;
+    background: #ffffff;
+    z-index: 2000;
+    padding: 0px 16px 200px 16px; /* extra top padding for logo & close */
+    overflow-y: auto;
+    box-sizing: border-box;
     display: flex;
```

Biome (2.1.2)

[error] Duplicate properties can lead to unexpected behavior and may override previous declarations unintentionally.

display is already defined here.

Remove or rename the duplicate property to ensure consistent styling.

(lint/suspicious/noDuplicateProperties)